payu/experiment.py

```diff
@@ -590,13 +593,6 @@ def run(self, *user_flags):
         if self.config.get('coredump', False):
             enable_core_dump()
 
-        # Our MVAPICH wrapper does not support working directories
-        if mpi_module.startswith('mvapich'):
-            curdir = os.getcwd()
-            os.chdir(self.work_path)
-        else:
-            curdir = None
-
```

Comment:
Just wanted to leave a comment to say deleting this relates directly to https://github.com/payu-org/payu/issues/350. 

This just can't work with submodels and there is no attempt to test for that, nor do we want to support something that is so limited, so it is being removed.